src/components/outlines/editor/index.js

```diff
@@ -1,27 +1,22 @@
+import React, { PropTypes } from 'react';
 import reactStamp from 'react-stamp';
+import EditorFactory from 'components/editor';
```

Comment:
I appreciate the effort here, but I should have been more clear: this file is garbage. The outlines section of the site will be discarded before the new editor goes live anywhere, so there's no need to refactor this. If you'd like, I can delete this on `master` and you can rebase (or you can `git rm` this directory and the associated route, but I know it wasn't part of the spec for this issue).